pkg/migrations/internal/seq_num.go

```diff
@@ -4,7 +4,7 @@ var (
 	// CurrentDBVersionSeqNum is the current DB version number.
 	// This must be incremented every time we write a migration.
 	// It is a shared constant between central and the migrator binary.
-	CurrentDBVersionSeqNum = 172
+	CurrentDBVersionSeqNum = 173
```

Comment:
first comment is I just pushed a change with migration 173, so sadly you will need to adjust that and your naming.

Reply:
https://www.youtube.com/watch?v=KpXsfimrkFo

Reply:
be warned @rhybrillou has a couple of migrations in flight too.